### PR TITLE
Improve viweable queueing

### PR DESCRIPTION
**Pull request by elland**
Partially solves #67. Could be a bit better.

### CODE REVIEW

File: Demo.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0800"
+   LastUpgradeVersion = "0810"
```
Comment (3lvis) [inline Demo.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme:3]: Get a proper Xcode version

Comment (3lvis) [reply Demo.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme:3]: Wait. 8.1 is App Store ready 😱

Comment (elland) [reply Demo.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme:3]: Take that!

File: Source/ViewerController.swift [hunk 2]
```diff
@@ -2,6 +2,18 @@ import UIKit
 import CoreData
 
 /**
+ Describe the index path as a simple string, so we can use it as a key for our queueing dictionary
+ */
+extension IndexPath {
+    var indexPathString: String {
+        get {
+            return String((self as NSIndexPath).section) + "-" + String((self as NSIndexPath).row)
```
Comment (3lvis) [inline Source/ViewerController.swift:10]: `self.section` should work as well.

Comment (elland) [reply Source/ViewerController.swift:10]: For this particular case, yes. But I just imported some code I had from `<redacted>`. 🙈

**Comment by 3lvis** [comment]
@Elland Can't wait to try it out! My Xcode is updating at the moment. Does it work with landscape?

**Comment by elland** [comment]
Works in landscape. But there's a rotation bug, not sure if it was there already? If you rotate with viewer up, the scroll direction lock turns off and you can move the images around a bit.

**Comment by 3lvis** [comment]
Had a look at this PR. The rotation issue is quite bad and wasn't there before. Would be great to get it fixed before merging this branch.

**Review by hemangshah — APPROVED** [review]

**Comment by 3lvis** [comment]
Closing because old, good idea, though.

**Pull request closed without merging**